quirks/password-rules.json

```diff
@@ -92,6 +92,9 @@
     "chase.com": {
         "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; required: [!#$%+/=@~]; max-consecutive: 2;"
     },
+    "cigna.co.uk": {
+        "password-rules": "minlength: 8; maxlength: 12; required: lower, upper, digit;"
```

Comment:
The screenshot makes it sound like their password rules require at least one number, at least one lowercase letter, and at least one uppercase letter, all at the same time. That requirement would need to be expressed with:

```
required: lower; required: upper; required: digit;
```

This ensures that at least one of each character type will be present (whereas `required: lower, upper, digit;` does not make that guarantee).

Reply:
Thanks, I didn't realize that difference. Updated now.